src/diffusers/__init__.py

```diff
@@ -146,6 +146,7 @@
             "RePaintScheduler",
             "SchedulerMixin",
             "ScoreSdeVeScheduler",
+            "ScoreSdeVpScheduler",
```

Comment:
cc @patrickvonplaten 

not sure why `ScoreSdeVpScheduler` is not currently in `__init__` - but if it is not currently in `__init__`, it means no one would use `from diffusers import ScoreSdeVpScheduler`, so we do not have to add it?

Reply:
I made the assumption that it was a forgotten import as `from diffusers import ScoreSdeVpScheduler` didn't work. If this is a mistake on my part, I'll revert back to the original.

Reply:
yeah I think we can just leave it as it is

Reply:
Alright, I'll revert back to the original. Thanks.

Reply:
@yiyixuxu Just to make sure, "by leaving it as is" do you mean to keep the current version or reverting back to the original version where the import here was not present?

Reply:
Alright, fixed! Thanks